evennia/contrib/mail.py

```diff
@@ -0,0 +1,194 @@
+"""
+In-Game Mail system
+
+Evennia Contribution - grungies1138 2016
+
+A simple Brandymail style @mail system that uses the Msg class from Evennia Core.
+
+Installation:
+    Import this module into the default Player or Character commandset.
+"""
+
+from evennia import default_cmds, search_object
+from evennia.utils import create, utils, evtable, evform
+from evennia.comms.models import Msg
+
+
+HEAD_CHAR = "|015-|n"
+SUB_HEAD_CHAR = "-"
+
+class MailCommand(default_cmds.MuxCommand):
+    """
+    Commands that allow either IC or OOC communications
+
+    Usage:
+        {w@mail{n
+                - Displays all the mail a player has in their mailbox
+
```

Comment:
Excellent docstring, but please make it in the style of the Evennia [Command documentation policy](https://github.com/evennia/evennia/wiki/Using%20MUX%20As%20a%20Standard#documentation-policy) - basically make a summary of the Switches and move the examples to the end. This is not just me being picky - doing so will help the API2md generator automatically create a nice API wiki page for the contrib.